api/reporting/schemas/latest/workspace.json

```diff
@@ -0,0 +1,143 @@
+[
+  {
+    "description": "Time snapshot was taken. Same across all rows in the snapshot, and can be used for partitioning.",
+    "name": "snapshot_timestamp",
+    "type": "INTEGER"
+  },
+  {
+    "name": "billing_account_type",
```

Comment:
These schemas are generated and regenerated with the script. This is fine for pre-release, but once we have a production dataset, there are many restrictions. For example, renaming or deleting columns isn't allowed.